microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -53,8 +53,40 @@ You must specify the full path of the file, for every resource.  For example:
 file:///C:/Users/username/Documents/GitHub/Demos/demo-to-do/index.html
 ```
 
+
+###### Cross-origin resources
+
 When specifying a file URL, the app navigates to a specific path, not a domain.  As a result, you can't use cross-origin resources for navigating to a file URL, because `location.origin` is `file://` and cross-origin rules are different for file URLs.
 
+
+###### Origin-based DOM APIs
+
+todo
```

Comment:
I could possibly add Supported or Not Supported next to each "todo" based on new table